handlers/pageData.go

```diff
@@ -126,13 +127,18 @@ func createMenuItems(active string) []types.MainMenuItem {
 			},
 		})
 	}
-	blockchainMenu = append(blockchainMenu, types.NavigationGroup{
+	clientsMenu = append(clientsMenu, types.NavigationGroup{
 		Links: []types.NavigationLink{
 			{
-				Label: "Clients",
-				Path:  "/clients",
+				Label: "Consensus",
+				Path:  "/clients/consensus",
 				Icon:  "fa-server",
 			},
+			{
+				Label: "Execution",
+				Path:  "/clients/execution",
+				Icon:  "fa-circle-nodes",
+			},
```

Comment:
can you add this menu point dynamically if at least 1 execution layer client is configured?
dora can be run without a execution layer configured (There are some details missing, but works in general).
we should not show pages that are known to not work at all if the configuration for it is missing (similar to the MEV Relays menu entry).